psl-core/src/test/java/org/linqs/psl/application/inference/MPEInferenceTest.java

```diff
@@ -195,4 +197,68 @@ public void testLogicalTautologyTrivial() {
         mpe.close();
         inferDB.close();
     }
+
+    /**
+    * Make sure that skipAtomCommit flag from CLI works and no atom is persisted in the database.
+    */
+    @Test
+    public void testSkipAtomCommit() {
+        // Arrange
+        TestModel.ModelInformation info = TestModel.getModel();
+        Set<StandardPredicate> toClose = new HashSet<StandardPredicate>();
+        Database inferDB = info.dataStore.getDatabase(info.targetPartition, toClose, info.observationPartition);
+        MPEInference mpe = new MPEInference(info.model, inferDB);
```

Comment:
This variable should just be an IA (the object can still be `MPEInference`).
Always make variables as general as they can be, e.g. you don't want to pass around an `ArrayList`, you want a `List` instead.
This is VERY important, talk to me if you don't understand.